worker/utils/utils.js

```diff
@@ -49,6 +51,27 @@ module.exports = {
     })
   },
 
+  async encryptJob(string1, string2){
+    const secret = this.retrievePassword() + string1 + string2
```

Comment:
If you parameterize salt the caller can generate it and then store it and use it to re-encrypt for the validation.